lib/publiccloud/instance.pm

```diff
@@ -919,9 +919,18 @@ sub do_systemd_analyze_time {
 
 sub upload_supportconfig_log {
     my ($self, %args) = @_;
-    $self->ssh_script_run(cmd => 'sudo supportconfig -R /var/tmp -B supportconfig -x AUDIT', timeout => 600);
-    $self->ssh_script_run(cmd => 'sudo chmod 0644 /var/tmp/scc_supportconfig.txz');
-    $self->upload_log('/var/tmp/scc_supportconfig.txz', failok => 1, timeout => 600);
+    return unless (get_var('SUPPORTCONFIG_LOG', 1));
+    # to exclude nothings, set "-"
+    my $exclude = get_var('SUPPORTCONFIG_EXCLUDE', 'AUDIT');
+    $exclude = ($exclude =~ s/[-\s]+//gr) ? '-x ' . $exclude : '';
+    my $log = "/var/tmp/scc_supportconfig";
+    my $cmd = "timeout --preserve-status 900 sudo supportconfig -R " . dirname($log) . " -B supportconfig $exclude";
```

Comment:
As we use this timeout twice, let's use a variable:

```suggestion
    my $timeout = 900;
    my $cmd = "timeout --preserve-status $timeout sudo supportconfig -R " . dirname($log) . " -B supportconfig $exclude";
```

Reply:
I already did it so before, in previous commits, but then I opted to reduce variables, to shorten the code. Anyway I agree